core/discovery/proposal/filter_test.go

```diff
@@ -19,12 +19,15 @@ package proposal
 
 import (
 	"testing"
+	"time"
 
 	"github.com/mysteriumnetwork/node/market"
 	"github.com/mysteriumnetwork/node/money"
 	"github.com/stretchr/testify/assert"
 )
 
+const bytesInGigabyte = 1000000000
```

Comment:
Calculation should be more accurate. There is some notion where you specify kilo as 1000, but you really imply 1024. That is, in reality you do all calculations with 1024 (cisco style). But i suggest to always use 1024 as  a kilo.. Ppl these days are smart enough to understand power of 2 calculations.

Reply:
moved to GiB instead of GB